src/ray/raylet/agent_manager.cc

```diff
@@ -58,8 +58,10 @@ void AgentManager::StartAgent() {
   }
   argv.push_back(NULL);
   // Set node id to agent.
+  static std::string pid_string = std::to_string(getpid());
   ProcessEnvironment env;
   env.insert({"RAY_NODE_ID", options_.node_id.Hex()});
+  env.insert({"RAY_NODE_PID", pid_string});
```

Comment:
Will this work if there are multiple raylets started on the same node? Could we pass these as command-line args instead?

Reply:
It's OK to start multiple raylets on the same node, but pass the `node id` and `ppid` as the command-line args is better. Let me fix this.